### PR TITLE
Added Blog Model

### LINKED ISSUE #142: Create Models for Blog page
### 🐣 Is your feature request related to a problem? Please describe.

Create Models for Blog page

### 📹 Screenshots or Video or GIF

---

**Note:**
* Do mention if you're working on this issue under ` LGM SOC 2021 `.
* If you want to work on any issue, you should check if it has already been assigned to anyone.

### PR DESCRIPTION
**Pull request by coding-geek21**
## What do these changes do?

added Blog Model

## Related Issue

Fixes #142 

## Checklist:

- [x] Have you checked there aren't other open [Pull Requests](https://github.com/Feminine-Divine/Woodog/pulls) for the same update/change?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video or GIF:

---

### CODE REVIEW

File: Woodog/Woodog/settings.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
     'woodogdata',
     'authentication',
     'appdashboard',
+    'froala_editor'
```
Comment (Aaishpra) [inline Woodog/Woodog/settings.py:46]: why this?

Comment (coding-geek21) [reply Woodog/Woodog/settings.py:46]: Using froala editor in django in place of the regular django Textarea editor we can get user input with, files and even videos

Comment (Aaishpra) [reply Woodog/Woodog/settings.py:46]: Okay

**Comment by coding-geek21** [comment]
@Aaishpra @khushishikhu Can you please review this PR

**Comment by khushishikhu** [comment]
@coding-geek21 please solve the merge conflicts

**Comment by coding-geek21** [comment]
> @coding-geek21 please solve the merge conflicts

sure i will do now

**Pull request merged**